stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/StartStopBuffer.kt

```diff
@@ -13,6 +13,7 @@ public class StartStopBuffer<T> {
     private val events: Queue<T> = ConcurrentLinkedQueue()
     private var active = AtomicBoolean(true)
     private var func: ((T) -> Unit)? = null
+    private var dataFilter: (T) -> T = { it }
```

Comment:
I think that `dataFilter` fits better inside `MessageLIstView`. A buffer normally doesn't filter data, it only controls when the data is sent

Reply:
You're right. Reworked solution